doc/radosgw/STS.rst

```diff
@@ -92,20 +92,28 @@ STS Configuration
 
 The following configurable options have to be added for STS integration::
 
-  [client.radosgw.gateway]
-  rgw sts key = {sts key for encrypting the session token}
-  rgw s3 auth use sts = true
+  [client.{your-rgw-name}]
+  rgw_sts_key = {sts key for encrypting the session token}
+  rgw_s3_auth_use_sts = true
 
-Note: By default, STS and S3 APIs co-exist in the same namespace, and both S3
-and STS APIs can be accessed via the same endpoint in Ceph Object Gateway.
+Notes: 
+
+* By default, STS and S3 APIs co-exist in the same namespace, and both S3
+  and STS APIs can be accessed via the same endpoint in Ceph Object Gateway.
+* The ``rgw_sts_key`` needs to be a hex-string consisting of exactly 16 characters.
 
 Examples
 ========
+1. In order to get the example to work, make sure that the user TESTER has both ``roles`` and ``user-policy`` capabilities assigned:
+
```

Comment:
The user-policy caps is needed only for GetSessionToken for STSLite, which is mentioned here: https://github.com/ceph/ceph/pull/46024/files under STS Lite REST APIs.

And the need for caps for roles has been mentioned here: https://docs.ceph.com/en/quincy/radosgw/role/ under REST APIS FOR MANIPULATING A ROLE.

But I agree its good to have these mentioned in one place, can you please add which caps is needed for which API, like user-policy for GetSessionToken and roles for AssumeRole* calls, to avoid further confusion.